gateway/single-cluster/global-l7-xlb/README.md

```diff
@@ -0,0 +1,526 @@
+
+The cluster `gke-1` is a regional cluster. The Gateway is hosted in default namespace. Two services in two different namespaces (other than default namespace). The Gateway (GKE load balancer) will match the traffic and send it to the right service depending on the request.
+
+This Recipe also demonstrates how to enable HTTPS on Gateway Load Balancer using Google-managed certificate.
+
+This example is using two applications; foo and bar. The *foo* applicaion is deployed in `NAMESPACE#1` and *bar* application is deployed in `NAMESPACE#2`. The External HTTPS Load Balancer is designed to route traffic to the services based on the request host name header. 
+
+The Gateway below also uses:
+
+- A [Google-Managed Certificate](https://cloud.google.com/kubernetes-engine/docs/how-to/deploying-gateways#tls_between_client_and_gateway)
+- A public [Static IP](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address) used as Gateway address
+
+```yaml
+kind: Gateway
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: external-http
+spec:
+  gatewayClassName: gke-l7-gxlb
+  listeners:
+  - protocol: HTTPS
+    port: 443
+    routes:
+      kind: HTTPRoute
+      namespaces:
+        from: "All"
+    tls:
+      mode: Terminate
+      options:
+        networking.gke.io/pre-shared-certs: gxlb-cert
+  addresses:
+  - type: NamedAddress
+    value: gke-gxlb-ip
+```
+
+The *spec.listeners.routes* in Gateway defines which routes can bind to the Gateway. In this example, routes from all namespaces are allowed. We can also use `selector` to restrict the routes binding to specific namespaces or services.
+
+The HTTPRoute resource defines how HTTP and HTTPS requests received by a Gateway are directed to Services. Application developers create HTTPRoutes to expose their applications through Gateways.
+
+
+```yaml
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: foo
+  namespace: gxlb-demo-ns1
+spec:
+  gateways:
+    allow: FromList
+    gatewayRefs:
+    - name: external-http
+      namespace: default
+  hostnames:
+  - "foo.$DOMAIN"
+  rules:
+  - forwardTo:
+    - serviceName: foo
+      port: 8080
+```
+
+The HTTPRoutes resources for two services (foo and bar) define which Gateway it can route traffic from (external-http in default namespace), which Services to route to (foo/bar), and rules that define what traffic the HTTPRoute matches (request host name is foo/bar.$DOMAIN). 
+
+Now that you have the background knowledge and understanding of GKE Gateway, you can try it out yourself.
+
+## Try it out
+
+1. Download this repo and navigate to this folder
+
+    ```bash
+    git clone https://github.com/GoogleCloudPlatform/gke-networking-recipes.git
+    Cloning into 'gke-networking-recipes'...
+
+    cd gke-networking-recipes/gateway/single-cluster/global-lb-basic
```

Comment:
```suggestion
    cd gke-networking-recipes/gateway/single-cluster/global-l7-xlb
```

Just stubled over it during using it for my recipe

Reply:
Thank you @gleichda! I corrected it.